Dockerfile

```diff
@@ -1,22 +1,43 @@
-FROM kbase/sdkbase2 as build
+FROM eclipse-temurin:11-jdk as build
+
+WORKDIR /tmp
+RUN apt-get update -y && \
+        apt-get install -y tzdata && apt-get install -y ca-certificates python3-sphinx ant git && \
+        git clone https://github.com/kbase/jars
 
 COPY . /tmp/workspace_deluxe
-RUN pip install configobj && \
-    cd /tmp && \
-    git clone https://github.com/kbase/jars && \
-    cd workspace_deluxe && \
+RUN cd workspace_deluxe && \
     make docker_deps
 
-FROM kbase/kb_jre
+# updated/slimmed down version of what's in kbase/kb_jre
+FROM ubuntu:18.04
```

Comment:
moved what was done on the kbase/kb_jre image into this Dockerfile

Reply:
Everything below is a straight C&P?

Reply:
Here's the [kb_jre image Dockerfile](https://github.com/kbase/kb_jre/blob/master/Dockerfile).

The only changes that I made were to turn the tomcat and dockerize versions into env vars for ease of updating.

Reply:
Looks like this line is missing?

```
update-ca-certificates && \
```

Reply:
Other than that yep, confirmed the same

Reply:
:+1: